### PR TITLE
[MasterPass] CAES-1600: Fix validation for confirm

### PR DESCRIPTION
**Pull request by sa715**
Signed-off-by: Sergey Alekseyev <sa@4xxi.com>

### CODE REVIEW

File: packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordConfirmForm.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import styled from 'styled-components';
-import { Formik, FastField } from 'formik';
+import { useFormik } from 'formik';
```
Comment (cu4xxi) [inline packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordConfirmForm.js:3]: 👍

File: packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordConfirmForm.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+import { useEffectOnce } from 'react-use';
 import { checkError } from '@caesar/common/utils/formikUtils';
 import {
   AuthTitle,
   AuthDescription,
   MasterPasswordInput,
   Button,
 } from '@caesar/components';
-import { createConfirmPasswordSchema } from './schema';
+import { createConfirmPasswordSchema } from "./schema";
```
Comment (cu4xxi) [inline packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordConfirmForm.js:12]: Don't you use auto-fixer by prettier, do you?

File: packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordConfirmForm.js [hunk 3]
```diff
@@ -30,43 +31,55 @@ const ButtonsWrapper = styled.div`
...
+    <Form onSubmit={handleSubmit}>
+      <AuthTitle>Confirmation</AuthTitle>
+      <AuthDescription>Confirm your master password</AuthDescription>
+      <MasterPasswordInput
+        name="confirmPassword"
+        value={values.confirmPassword}
+        autoFocus
+        isAlwaysVisibleIcon
+        placeholder="Repeat password…"
+        error={dirty ? checkError(touched, errors, 'confirmPassword') : null}
```
Comment (cu4xxi) [inline packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordConfirmForm.js:69]: hm, usually we use just checkError and show error when field was touched. 
Dirty is about all form.

File: packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordConfirmForm.js (hunk 2 above)
Comment (sa715) [reply packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordConfirmForm.js:12]: Yes, autoimport from WS

File: packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordConfirmForm.js (hunk 3 above)
Comment (sa715) [reply packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordConfirmForm.js]: It was transfered from MasterPasswordCreateForm. Removed condition for dirty

**Review by rtatarinov4xxi — APPROVED** [review]

**Review by cu4xxi — APPROVED** [review]

**Pull request merged**